snuba/snuba_migrations/events_analytics_platform/0041_hashed_keys_index.py

```diff
@@ -0,0 +1,46 @@
+from typing import Sequence
+
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+
+buckets = 40
+
+
+def get_hashed_keys_index_expression() -> str:
+    column_expressions = []
+    for i in range(buckets):
+        hashed_keys = f"arrayMap(k -> cityHash64(k), mapKeys(attributes_string_{i}))"
+        column_expressions.append(hashed_keys)
+
+    return f"arrayConcat({', '.join(column_expressions)})"
+
+
+class Migration(migration.ClickhouseNodeMigration):
+
+    blocking = False
+    storage_set_key = StorageSetKey.EVENTS_ANALYTICS_PLATFORM
+    granularity = "8192"
+
+    local_table_name = "eap_items_1_local"
+
+    def forwards_ops(self) -> Sequence[operations.SqlOperation]:
```

Comment:
have you tested locally that this bloom filter index would actually be hit? I don't know how clickhouse would even know to use it